examples/failure_injection.py

```diff
@@ -12,13 +12,13 @@ async def run():
     print("Waiting for drone to connect...")
     async for state in drone.core.connection_state():
         if state.is_connected:
-            print(f"Drone discovered with UUID: {state.uuid}")
+            print(f"-- Connected to drone!")
```

Comment:
If I recall correctly, `{state.uuid}` was causing issues with some MAVSDK versions. At least I got errors sometimes trying to run this script. Should we just remove it or is there a new "proper way" to access the UUID?

Reply:
Now the UUID is part of the info plugin, I think :+1: